packages/terra-card/tests/wdio/card-spec.js

```diff
@@ -0,0 +1,53 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny');
+
+const shouldTheme = (customProperties) => {
+  Object.entries(customProperties).forEach(([key, value]) => {
+    it(`themed [${key}]`, () => {
+      browser.setCSSCustomProps({ [key]: value });
+      expect(browser.checkElement('[data-reactroot]')).to.matchReference();
+    });
+  });
+};
+
+describe('Badge', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/tests/card-tests/default'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    shouldTheme({
+      '--terra-card-background-color': 'yellow',
+      '--terra-card-border': '10px dashed purple',
+      '--terra-card-border-radius': '50px',
```

Comment:
It looks applying a  border-radius cuts the text off when looking at the screenshot. Did we get an issue logged to fix this styling bug?

Reply:
Well.... its standard browser behavior. The big question is if we want to implement something to prevent it. It would be kinda tricky IMO with theming... meaning the theme would just have to account for its border radius and make sure text isn't clipped @bjankord  thoughts?

Reply:
I think we leave it as is, my hope would be if people apply 50px border-radius, they also apply enough padding to fix clipping issues which result from the larger border-radius.

Reply:
Either sounds good to me!